SemanticComparisonComment.md

```diff
@@ -0,0 +1,3 @@
+Hi Mark,
+
+I wonder where I can have a look at the source code for the `SemanticComparison` package (if possible)
```

Comment:
It's at https://github.com/AutoFixture/AutoFixture/tree/master/Src/SemanticComparison